frontend/public/components/modals/expandPvc-modal.jsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory/modal';
+import { PromiseComponent, RequestSizeInput } from '../utils';
+import { k8sUpdate } from '../../module/k8s/';
+import { PersistentVolumeClaimModel } from '../../models';
+
+//Modal for expanding persistent volume claims
+class ExpandPVCModal extends PromiseComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inProgress: false,
+      errorMessage: '',
+      requestSizeValue: '',
+      requestSizeUnit: 'Gi',
+      pvcObj: null,
+      validationSuccessful: false,
+      dropdownUnits: {
+        Mi: 'Mi',
+        Gi: 'Gi',
+        Ti: 'Ti',
+      },
+    };
+    this._handleRequestSizeInputChange = this._handleRequestSizeInputChange.bind(this);
+    this._cancel = this.props.cancel.bind(this);
+    this._submit = this._submit.bind(this);
+    this.updateSize = this.updateSize.bind(this);
+  }
+
+  _handleRequestSizeInputChange(obj) {
+    this.setState({ requestSizeValue: obj.value, requestSizeUnit: obj.unit }, this.updateSize);
+  }
+
+  updateSize() {
+    const { resource } = this.props;
+    const { requestSizeValue, requestSizeUnit } = this.state;
+    const obj = {
+      ...resource,
+      spec: {
+        ...resource.spec,
+        resources: {
+          requests: {
+            storage: `${requestSizeValue}${requestSizeUnit}`,
+          },
+        },
+      },
+    };
+    this.setState({pvcObj: obj, validationSuccessful: true});
+  }
+
+  _submit(e) {
+    e.preventDefault();
+    this.handlePromise(k8sUpdate(PersistentVolumeClaimModel,this.state.pvcObj))
+      .then(this.props.close)
+      .catch(this.setState({validationSuccessful: false}));
+  }
+
+  render() {
+    const {kind, resource} = this.props;
+    const { requestSizeUnit, requestSizeValue, dropdownUnits } =this.state;
+    return <form onSubmit={this._submit} name="form" className="modal-content modal-content--no-inner-scroll">
+      <ModalTitle>Expand {kind.label}</ModalTitle>
+      <ModalBody className="modal-body">
+        <div className="form-group">
+          <p className="lead">Increase the capacity of claim <strong>{resource.metadata.name}.</strong></p>
```

Comment:
`<p className="co-help-text">This can be a time-consuming process.</p>`
is it ok ?
![help](https://user-images.githubusercontent.com/25664409/57325901-ce94ce80-7128-11e9-88c9-b7ce2b6d4a59.png)